selfdrive/loggerd/logger.h

```diff
@@ -53,8 +54,12 @@ class BZFile {
   BZFILE* bz_file = nullptr;
 };
 
+typedef cereal::Sentinel::SentinelType SentinelType;
+
 typedef struct LoggerHandle {
   pthread_mutex_t lock;
+  SentinelType end_sentinel_type;
+  bool exit_signal;
```

Comment:
exit signal is not a bool. We want to distinguish SIGINT from SIGPWR.

Reply:
fixed